src/services/axios.ts

```
@@ -28,123 +30,69 @@ axiosInstance.interceptors.request.use(
+    return;
+  }
+
+  try {
+    console.log('Attempting scheduled token refresh...');
+    const refreshResponse = await axios.post(API_BASE_URL + REFRESH_TOKEN_URL, {
+      refreshToken: currentRefreshToken,
+    });
+
+    const { accessToken: newAccessToken, refreshToken: newRefreshToken } = refreshResponse.data;
+    authStore.SetTokens(newAccessToken, newRefreshToken || currentRefreshToken);
+    axiosInstance.defaults.headers.common['Authorization'] = 'Bearer ' + newAccessToken;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)